### ISSUE TITLE
Feat(eos_cli_config_gen): support the same system.control_plane.ipxx_access_groups ACL on multiple VRFs

### ISSUE THREAD

**Issue by philippebureau** [aristanetworks/ansible-avd#4077]
### Enhancement summary

currently AVD expect a unique value for system.control_plane.ipxx_access_groups.acl_name
If the same ACL is used in multiple VRFs, AVD throw a validation error : 
```Validation Error: system.control_plane.ipv4_access_groups[0].acl_name': The value 'test' is not unique between all list items as required.```

### Which component of AVD is impacted

eos_cli_config_gen

### Use case example

use the same control plane ACL for multiple VRFs

### Describe the solution you would like

```yaml
custom_structured_configuration_system:
  control_plane:
    ipv4_access_groups:
      - acl_name: test
      - acl_name: test
        vrf: MGMT
```

### Describe alternatives you have considered

should the data model be vrf based?
```
system:
  control_plane:
    vrfs:
      - name: default
            ipv4_access_groups:
              acl_name: <str; required; unique>
      - name: MGMT
            ipv4_access_groups:
              acl_name: <str; required; unique>
```

### Additional context

The current implementation works as it produces the correct output but it throws a lot of warnings in the logs

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

**Comment by BouchardClaude** [comment]
Hello. 
This feature is preventing going into blocking mode on build. Is there any known workaround ?

thanks

**Comment by carlbuchmann** [comment]
Fixed in AVD 5.0.0 and now get the following output:

```eos
system control-plane
   ip access-group test in
   ip access-group test vrf MGMT in
```
